### PR TITLE
[enriched/github] Fix KeyError when getting user info

### PR DESCRIPTION
**Pull request by zhquan**
Perceval return this dict when the user is no longer exist or the
token doesn't have permission.
```
{"organizations": []}
```

User parameters are set to 'None' on this change.

Signed-off-by: Quan Zhou <quan@bitergia.com>

### CODE REVIEW

File: grimoire_elk/enriched/github.py [hunk 1]
```diff
@@ -51,6 +51,10 @@
 
 GITHUB = 'https://github.com/'
 
+# Perceval return this dict when the user is no longer exist
+# or the token has no permission
```
Comment (sanacl) [inline grimoire_elk/enriched/github.py:55]: It should be

``` # Perceval returns this dict when the user no longer exists```

File: grimoire_elk/enriched/github.py [hunk 2]
```diff
@@ -548,7 +550,7 @@ def __get_rich_issue(self, item):
         rich_issue['user_login'] = issue['user']['login']
 
         user = issue.get('user_data', None)
-        if user is not None and user:
+        if user is not None and user and user != NO_USER_INFO:
```
Comment (sanacl) [inline grimoire_elk/enriched/github.py:553]: why don't you create a method to evaluate whether user is available or not? this way the code is more readable, don't you think?

**Review by canasdiaz — CHANGES_REQUESTED** [review]
One grammatical fix and a **suggestion** to make code more readable. If you don't like the suggestion I have nothing to say, my python skills are not in a good shape ;)

**Review by canasdiaz — APPROVED** [review]

**Comment by sanacl** [comment]
@zhquan LGTM

**Pull request merged**